src/data-pipeline/spark-etl/src/main/resources/data_schema.json

```diff
@@ -151,6 +151,12 @@
       "nullable": true,
       "metadata": {}
     },
+    {
```

Comment:
Did you test upgrading from v1.0.0 to latest version with the schema changes?

Reply:
yes. test upgroup from v1.0.0, no issue, event with existing files, the new added filed will be `null` for existing data.